### ISSUE TITLE
Sonoff ZBMINIR2 does not work in Detach relay mode in Dev, works in stable

### ISSUE THREAD

**Issue by olivercjung** [Koenkk/zigbee2mqtt#25146]
### What happened?

In the current dev version, Sonoff ZBMINIR2 does not work in _Detach relay mode_. It works in the stable version. 

### What did you expect to happen?

When the switch is toggled in Detach relay mode, an Action message should be generated which could be used in automations. In the stable version this is working. In current dev version, it is not. There is no Action message generated.

### How to reproduce it (minimal and precise)

Put Soonoff ZBMINIR2 in _Detach relay mode_. Toggle the switch. No message is generated.

### Zigbee2MQTT version

1.42.0-dev commit: 7803a16

### Adapter firmware version

7.4.4 [GA]

### Adapter

SONOFF ZBDongle-E

### Setup

Add-On Home Assistant, running in Proxmox VM

### Debug log

Since no message is generated, there is no debug log.

**Comment by olivercjung** [comment]
This still does not seem to work in Dev. Is there anything I could do to support?

**Comment by WillBeasley** [comment]
Hi, Just wanted to add that I have had a similar problem happen where I was using 1.42.0 commit 861cba6 via docker with no issues until I pulled latest yesterday and the toggle Action became unavailable in HomeAssistant. 

Only by reverting back to the older zigbee2mqtt did I solve this issue.

**Comment by olivercjung** [comment]
This still does not work and I had to go back to 1.42.0 again. 

Is there any plan to make detach mode work again in 2.x?

**Comment by olivercjung** [comment]
This is the debug log:

```
info 2025-02-19 07:55:17z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Lichtschalter Küche', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":21,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":156,"network_indicator":null,"power_on_behavior":"previous","state":"ON","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"},"update_available":false}'
info 2025-02-19 07:55:17z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Lichtschalter Küche/action', payload 'toggle'
info 2025-02-19 07:55:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Lichtschalter Küche', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":21,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":156,"network_indicator":null,"power_on_behavior":"previous","state":"ON","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"},"update_available":false}'
info 2025-02-19 07:55:21z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Lichtschalter Küche/action', payload 'toggle'
```

Here it seems to work, but Home Assistant does not receive any updates.

**Comment by alex93133** [comment]
Any updates?

**Comment by vitalybibikov** [comment]
Hey, Can I help somehow for it being fixed? If I can provide info please let me know

Logs from the device being switched on/off in detached relay mode:
`
info 2025-06-07 13:14:31z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":0.5,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":22,"network_indicator":true,"power_on_behavior":"previous","state":"OFF","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"}}'
info 2025-06-07 13:14:31z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch/action', payload 'toggle'
info 2025-06-07 13:14:31z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-bed-wall-led-right-double-switch', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":0.5,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":50,"network_indicator":true,"power_on_behavior":"previous","state":"ON","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"}}'
info 2025-06-07 13:14:31z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-bed-wall-led-right-double-switch/action', payload 'toggle'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":0.5,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":45,"network_indicator":true,"power_on_behavior":"previous","state":"OFF","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"}}'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch/action', payload 'toggle'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-bed-wall-led-right-double-switch', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":0.5,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":53,"network_indicator":true,"power_on_behavior":"previous","state":"ON","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"}}'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-bed-wall-led-right-double-switch/action', payload 'toggle'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch', payload '{"action":"toggle","delayed_power_on_state":false,"delayed_power_on_time":0.5,"detach_relay_mode":true,"external_trigger_mode":"edge","linkquality":40,"network_indicator":true,"power_on_behavior":"previous","state":"OFF","turbo_mode":false,"update":{"installed_version":4100,"latest_version":4100,"state":"idle"}}'
info 2025-06-07 13:14:32z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/bedroom-window-led-bed-right-double-switch/action', payload 'toggle'
info 2025-06
`


So basically it sends 

 topic 'zigbee2mqtt/bedroom-bed-wall-led-right-double-switch/action', payload 'toggle'

**Comment by vitalybibikov** [comment]
Hmm, it does, work, I was listening to the a bit incorrect topic.

So basically, for the device described above, the following automation should work:
```

alias: Toggle LED strip from double switch
description: Toggles LED strip when the switch sends a toggle payload
triggers:
  - topic: zigbee2mqtt/bedroom-bed-wall-led-right-double-switch/action
    payload: toggle
    trigger: mqtt
conditions: []
actions:
  - action: light.toggle
    data: {}
    target:
      entity_id: light.bedroom_bed_wall_led
mode: single

```
